src/webapp/components/landing-page-edit-dialog/LandingPageEditDialog.tsx

```diff
@@ -27,16 +28,26 @@ const buildDefaultNode = (type: LandingNodeType, parent: string, order: number)
         content: undefined,
         children: [],
         modules: [],
+        permissions: {
+            publicAccess: "r-------",
+            userAccesses: [],
+            userGroupAccesses: [],
+        },
+        executeOnInit: executeOnInit,
     };
 };
```

Comment:
Now that we are touching the object, let's review it. The function should have an explicit return type. And probably the full object can be defined in the entity, with a good name of what represents.